test/mod/test_fmt_v2.py

```diff
@@ -72,8 +72,41 @@
     ]
 }
 
+BAD_SHA = "sha256:d540ca8c5e21ba5f063286c94a088af2aac0b15bc40df6fd562d40154c10f4a1"
+
+BAD_REF_PIPELINE = {
+  "version": "2",
+  "sources": {
+    "org.osbuild.curl": {
+      "items": {
+        "sha256:c540ca8c5e21ba5f063286c94a088af2aac0b15bc40df6fd562d40154c10f4a1": "",
+      }
+    }
+  },
+  "pipelines": [
+    {
+      "name": "build",
+      "stages": [
+        {
+          "type": "org.osbuild.rpm",
+          "inputs": {
+            "packages": {
+              "type": "org.osbuild.files",
+              "origin": "org.osbuild.source",
+              "references": {
+                BAD_SHA: {},
```

Comment:
Isn't `BAD_SHA` actually the good one? Because the other one (15a65) is not defined in sources.

Reply:
Good catch thanks!